packages/mask/src/plugins/Avatar/Services/rss3.ts

```diff
@@ -23,6 +23,11 @@ export async function getNFTAvatarFromRSS(userId: string, address: string, snsKe
     return result?.nft
 }
 
+export function clearTargetCache(userId: string, address: string, snsKey: RSS3_KEY_SNS) {
```

Comment:
`clear` seems you'd like to delete all cached data.

```suggestion
export function deleteTargetCache(userId: string, address: string, snsKey: RSS3_KEY_SNS) {
```